src/traffic_layout/CMakeLists.txt

```diff
@@ -34,10 +34,6 @@ target_link_libraries(traffic_layout
         ${OPENSSL_LIBRARY}
 )
 
-if(TS_USE_HWLOC)
-    target_link_libraries(traffic_layout PRIVATE hwloc::hwloc)
-endif()
-
```

Comment:
info.cc needs to get the API version from hwloc if it's enabled, so hwloc is a usage requirement.